charts/kube-prometheus-stack/templates/grafana/dashboards-1.14/k8s-resources-cluster.yaml

```diff
@@ -247,7 +247,7 @@ data:
                         "steppedLine": false,
                         "targets": [
                             {
-                                "expr": "sum(kube_pod_container_resource_limits_cpu_cores{cluster=\"$cluster\"}) / sum(kube_node_status_allocatable_cpu_cores{cluster=\"$cluster\"})",
+                                "expr": "sum(kube_pod_container_resource_limits{cluster=\"$cluster\", resource=\"cpu\"}) / sum(kube_node_status_allocatable{resource=\"cpu\",cluster=\"$cluster\"})",
```

Comment:
this is requiring kube-state-metrics 2.0 right?
See https://github.com/prometheus-community/helm-charts/pull/951#issuecomment-836707944

Reply:
it should not, only kube-state-metrics 2.0 removes the deprecated metric `kube_pod_container_resource_limits_cpu_cores` as `kube_pod_container_resource_limits` was introduced in a previous version.